plugins/spi/SPIWriter.cpp

```diff
@@ -18,6 +18,7 @@
  * Copyright (C) 2013 Simon Newton
  */
 
+#include <linux/ioctl.h>
```

Comment:
Can you put this above linux/spi/spidev.h please, unless it has to be right at the top for some reason?